docs/index.rst

```diff
@@ -0,0 +1,70 @@
+Bayesian Tracker (btrack) 🔬💻's
+================================
+
+|logo|
+
+BayesianTracker (``btrack``) is a Python library for multi object tracking, used to reconstruct trajectories in crowded fields.
+Here, we use a probabilistic network of information to perform the trajectory linking.
+This method uses spatial information as well as appearance information for track linking.
+
+The tracking algorithm assembles reliable sections of track that do not contain splitting events (tracklets).
+Each new tracklet initiates a probabilistic model, and utilises this to predict future states (and error in states) of each of the objects in the field of view.
+We assign new observations to the growing tracklets (linking) by evaluating the posterior probability of each potential linkage from a Bayesian belief matrix for all possible linkages.
+
+The tracklets are then assembled into tracks by using multiple hypothesis testing and integer programming to identify a globally optimal solution.
+The likelihood of each hypothesis is calculated for some or all of the tracklets based on heuristics.
+The global solution identifies a sequence of high-likelihood hypotheses that
+accounts for all observations.
```

Comment:
`nit`: wrap at 80 char

Reply:
yeah hard to say what is best. For my thesis I did one line per sentence. But usually for `markdown` etc I tend to do 80 🤷

Reply:
Changed all (`md` and `rst`) to [semantic linefeed](https://rhodesmill.org/brandon/2012/one-sentence-per-line/). However we should aim to keep sentences short :wink: